.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 ## Description
-Provide a brief description of the PR's purpose here. If your PR is supposed to fix an existing issue, use
-a [keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) to link your PR to the issue.
+<!--Provide a brief description of the PR's purpose here. If your PR is supposed to fix an existing issue, use
```

Comment:
Is there a reason we are flanking these with the `<!--` HTML comment syntax? Might just not be aware of why this change is being made.

Reply:
I wanted to hide the suggestions in `.md` format. Otherwise, if a PR is created, it shows up like this:

![image](https://github.com/user-attachments/assets/bd6cbd15-721d-4876-9d5b-f953987637e4)

With the newer changes, the instructions are still available if a user wants them but they don't clutter the description box.
![image](https://github.com/user-attachments/assets/5da421f1-61ea-4480-b9a1-d8504f310599)

Reply:
Ah okay. In that case, everything else seems fine, LGTM! 🚀